components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/dao/OrganizationManagementDAO.java

```diff
@@ -409,4 +409,24 @@ String getAnAncestorOrganizationIdInGivenDepth(String organizationId, int depth)
      * @throws OrganizationManagementServerException The server exception thrown when adding a root organization.
      */
     void addRootOrganization(Organization rootOrganization) throws OrganizationManagementServerException;
+
+    /**
+     * Retrieve the list of organizations' meta attributes.
+     *
+     * @param recursive               Determines whether records should be retrieved in a recursive manner.
+     * @param limit                   The maximum number of records to be returned.
+     * @param organizationId          The super organization ID.
```

Comment:
Is this the root org always? or else the org which the request is initiated?

Reply:
It is always the root organization of that tenant domain.